src/derivatives/atlas.md

```diff
@@ -357,7 +354,10 @@ A guide for using macros can be found at
 
 For example, the [PS13 template](https://doi.org/10.18112/openneuro.ds004401.v1.3.0),
 a molecular imaging brain template of Cyclooxygenase-1 (PET),
-was generated in two standard spaces: `MNI152Lin` and `fsaverage`:
+was generated in two standard spaces: `MNI152Lin` and `fsaverage`.  Here, the use of 
```

Comment:
I've reviewed up to this point. I will stop here and let's continue together on a call :)

Reply:
Thanks for the review, @oesteban.  I think a lot of my changes come from my misunderstanding of the intent of the proposal.  Looking forward to discussing and clarifying tomorrow.